src/me/StevenLawson/TotalFreedomMod/Commands/Command_invis.java

```diff
@@ -19,7 +19,7 @@ public boolean run(CommandSender sender, Player sender_p, Command cmd, String co
     {
         boolean smite = false;
         if (args.length >= 1)
-        {
+        {   
```

Comment:
@rsajeey Formatting errors can be down to using the default formatting settings in Netbeans... Not because they never wrote it in an IDE.